src/NzbDrone.Core/HealthCheck/Checks/DownloadClientSortingCheck.cs

```diff
@@ -1,5 +1,4 @@
 using System;
-using System.Linq;
```

Comment:
All these using clean ups should be part of separate commit, possibly even a separate PR.

Reply:
@mynameisbogdan did these in a separate commit, they can be removed.